appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java

```diff
@@ -64,23 +65,11 @@ public SimpleVisibleComponentsPanel(final SimpleEditor editor,
 
     checkboxShowHiddenComponents = new CheckBox(MESSAGES.showHiddenComponentsCheckbox()) {
       @Override
-      protected void onLoad() {
-        // onLoad is called immediately after a widget becomes attached to the browser's document.
-        boolean showHiddenComponents = Boolean.parseBoolean(
-            projectEditor.getProjectSettingsProperty(
-            SettingsConstants.PROJECT_YOUNG_ANDROID_SETTINGS,
-            SettingsConstants.YOUNG_ANDROID_SETTINGS_SHOW_HIDDEN_COMPONENTS));
-        checkboxShowHiddenComponents.setValue(showHiddenComponents);
-      }
+      protected void onLoad() {}
     };
     checkboxShowHiddenComponents.addValueChangeHandler(new ValueChangeHandler<Boolean>() {
       @Override
       public void onValueChange(ValueChangeEvent<Boolean> event) {
```

Comment:
One problem with removing this functionality is that we will no longer save state between sessions.